etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/PostgresDao.java

```diff
@@ -121,11 +129,18 @@ public synchronized PartnerMetadata fetchMetadata(String receivedSubmissionId)
                 return null;
             }
 
+            Instant timeReceived = null;
+            Timestamp timestamp = result.getTimestamp("time_received");
+            if (timestamp != null) {
+                timeReceived = timestamp.toInstant();
+            }
+
```

Comment:
If timestamp is null this will still throw a null pointer

Reply:
Correct, but that statement is wrapped in that `if` statement that ensures that `timestamp` is _not_ `null`.